packages/coil-extension/src/background/consts/extensionInstances.ts

```diff
@@ -0,0 +1,26 @@
+// add the id's of the extensions to check their active states
+export interface ExtensionInstance {
+  extensionName: string
+  extensionId: string
+}
+
+export type BrowserType = 'chrome' | 'firefox' | 'edge'
+
+export const EXTENSIONS: Record<BrowserType, Array<ExtensionInstance>> = {
+  chrome: [
+    {
+      extensionName: 'CoilDevDist2',
+      extensionId: 'deofcdfbmgdlnbdhpdmmegpokkaoadam'
+    },
+    {
+      extensionName: 'CoilDev',
+      extensionId: 'hcohoecolgmlofifjaobjhidpoaciknp'
+    },
+    {
+      extensionName: 'CoilPreview',
+      extensionId: 'iehmfkldnblennopinmmagfidpflefkp'
+    }
+  ],
+  firefox: [],
```

Comment:
For Firefox:
https://addons.mozilla.org/en-US/firefox/addon/coil/

For edge:
https://microsoftedge.microsoft.com/addons/detail/coil/ljionajlbinlfkdnpkloejeoogfgkojm